classes/pdf/HTMLTemplate.php

```diff
@@ -123,7 +123,7 @@ public function assignCommonHeaderData()
         $width = 0;
         $height = 0;
         if (!empty($path_logo)) {
-            list($width, $height) = getimagesize($path_logo);
+            list($width, $height) = getimagesize(_PS_ROOT_DIR_.$path_logo);
```

Comment:
This could work, but if I refer to https://github.com/PrestaShop/PrestaShop/pull/9395/files#r207572067, we cannot accept this change as this method could break the retro compatibility for modules relying on it.

We must find something else which does not imply to concatenate data to the returned sting. It must be done inside the method and still work here. See my other comment for another fix.

Reply:
Reverting this line and accepting the other suggestion would fix your issue and the dimension problem.